sources/packages/forms/src/form-definitions/sfaa2023-24.json

```diff
@@ -1534,16 +1535,7 @@
               "multiple": false,
               "dataSrc": "values",
               "data": {
-                "values": [
```

Comment:
Other forms doesn't have this change. Please make sure all 3 SFAA* forms have same content except title, name and path